src/css/components/menu.css

```diff
@@ -15,28 +15,10 @@
 
 .menu__link {
   font-size: var(--font-size-sm);
-  color: var(--gray10);
   transition: all 0.3s ease-in-out;
```

Comment:
**suggestion**:
(I hope this is the right place)

IMO, we should decrease the duration of the transition. Moving cursor from top of the sidebar item list to the bottom leaves a background color "smudge" for too long (to my taste).


https://github.com/saleor/saleor-docs/assets/44495184/6cf4319b-57a1-4b97-9a46-6c6b745785ba

Reply:
This is completely unrelated to this PR, though. I only reverted even uglier sidebar styling to its original state.

Reply:
Current docs don't have group pages, so we would need to solve it somehow

Reply:
I disagree it's unrelated. You changed the way the sidebar looks and I was just sharing my opinion that this particular bit, to me, now looks off.

Reply:
I removed all easing from the sidebar, so the animation is gone entirely.